lib/bitbucket_rest_api/repos.rb

```diff
@@ -10,7 +10,10 @@ class Repos < API
                  :Keys        => 'keys',
                  :Services    => 'services',
                  :Following   => 'following',
-                 :Sources     => 'sources'
+                 :Sources     => 'sources',
+                 :Forks       => 'forks',
+    :Commits =>'commits',
+    :Download=>'download'
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.